portal/src/pages/demoer.tsx

```diff
@@ -0,0 +1,58 @@
+import { NavCard } from "@fremtind/jkl-card-react";
+import { motion } from "framer-motion";
+import { Link } from "gatsby";
+import React, { FC } from "react";
+import { Seo } from "../components/seo";
+import "../styles/demoer.scss";
+
+export const Head: FC = () => <Seo title="Demoer" />;
+
+const DemoPage: FC = () => (
+    <motion.main
+        initial={{ opacity: 0 }}
+        animate={{ opacity: 1 }}
+        exit={{ opacity: 0 }}
+        transition={{ duration: 0.35 }}
+        className="jkl-portal__main jkl-portal__main--full-width"
+    >
+        <h1 className="jkl-title jkl-spacing-2xl--bottom jkl-spacing-2xl--top">Demoer</h1>
+        <p className="jkl-portal-paragraph">
+            Demoer viser eksempler på bruk av komponenter i en større sammenheng. Hvert eksempel opner seg i en egen
+            fane.
+        </p>
+        <ul className="jkl-portal-demoer__demo-list">
+            <li className="jkl-portal-demoer__demo">
+                <NavCard
+                    component={Link}
+                    padding="l"
+                    to="/demoer/skjemavalidering"
+                    title="Skjemavalidering"
+                    image={{ src: "/assets/demoer/skjemavalidering.png", alt: "" }}
```

Comment:
Her tror jeg du må (eller bør) importere bildene i toppen av siden, og heller bruke importen her. Sånn det er nå blir stien feil i preview-branchen, og potensielt også i prod. Se f.eks. [NavCardExample](https://github.com/fremtind/jokul/blob/main/packages/card-react/documentation/NavCardExample.tsx) for et eksempel 😊

Reply:
Aha! Det skal bli riktig i prod siden det (som localhost) kjører på `/` som basePath. Men godt tips – tenkte ikke på å sjekke der.